simulate-event.js

```diff
@@ -1,20 +1,19 @@
-var extend = require('xtend')
 
 /**
  * Set some default options.
  *
  * @type {Object}
  */
 var eventOptions = {
-  UIEvent: function (el) {
+  UIEvent: function () {
     return {
-      view: el.ownerDocument.defaultView
+      view: document.defaultView
```

Comment:
I don't follow how it can be passed in, as another optional argument after options?